homeassistant/components/whirlpool/config_flow.py

```diff
@@ -27,6 +26,7 @@
         vol.Required(CONF_USERNAME): str,
         vol.Required(CONF_PASSWORD): str,
         vol.Required(CONF_REGION): vol.In(list(CONF_REGIONS_MAP)),
+        vol.Required(CONF_BRAND): vol.In(list(CONF_BRANDS_MAP)),
```

Comment:
The translation strings in `strings.json` haven't been updated. Please do that. That will also show how we're going to explain this to the user, which is important.

Reply:
I've added a line in `string.json` for the brand. Whirlpool makes the brand very confusing, as far as I can tell it only affects the output of the shared appliances query, so the string I added was `"brand": "Brand of appliances"`. I'm not sure what happens if there are users with shared appliances of multiple brands, but I think we'll have to cross that bridge when we come to it.